src/runtime/internal/lifecycle.ts

```diff
@@ -27,22 +27,33 @@ export function onDestroy(fn: () => any) {
 	get_current_component().$$.on_destroy.push(fn);
 }
 
-export function createEventDispatcher<
-	EventMap extends {} = any
->(): <EventKey extends Extract<keyof EventMap, string>>(type: EventKey, detail?: EventMap[EventKey]) => void {
+interface DispatchOptions {
```

Comment:
Since it's part of the public function signature I would also export this

Reply:
Exported.